api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -2149,5 +2152,29 @@ public static boolean isYesterday(Date date) {
 	public static Set<String> getDeclaredFields(Class<?> clazz) {
 		return Arrays.stream(clazz.getDeclaredFields()).map(Field::getName).collect(Collectors.toSet());
 	}
+
+	/**
+	 * Checks if a given value is within the normal reference range for a concept and person.
+	 *
+	 * @param value The value to check
+	 * @param concept The concept associated with the value
+	 * @param person The person associated with the observation
+	 * @return Error message
```

Comment:
Can you improve this documentation such that whoever used this function knows what to expect?